flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -153,6 +153,30 @@
 		.defaultValue(false)
 		.withDescription("Monitor the current actual delayed write rate. 0 means no delay.");
 
+	public static final ConfigOption<Boolean> IS_WRITE_STOPPED = ConfigOptions
+		.key(RocksDBProperty.IsWriteStopped.getConfigKey())
+		.booleanType()
+		.defaultValue(false)
+		.withDescription("Track whether write has been stopped in RocksDB. Returns 1 if write has been stopped, 0 otherwise.");
```

Comment:
Isn't the return value a RocksDB detail? Why are we documenting that?

Reply:
IMO it could help understand the data of this metrics and how to monitor/read it, following the same pattern as [RocksDBProperty.CompactionPending](https://github.com/apache/flink/blob/65853b375abd1e6b3d2b9c7734d7f93aa5d529a0/flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java#L57)

Reply:
I think a brief explanation would be helpful for users.

Reply:
@zentol What's your thought of my reply and consider to take a look at this PR again.